Resources/Other-Sources/en.lproj/InfoPlist.strings

```diff
@@ -1,2 +1,3 @@
 "CFBundleDisplayName" = "Tropos";
 "NSLocationWhenInUseUsageDescription" = "Tropos will only use your location to discover current weather conditions and forecasts for your area.";
+"NSLocationAlwaysUsageDescription" = "Tropos will only use your location stay updated on current weather conditions and forecasts for your area.";
```

Comment:
Good catch. Added in https://github.com/thoughtbot/Tropos/commit/ae34fc5121ffd07560e2d2389a4e03ad5edfc403